cfme/ansible/repositories.py

```diff
@@ -256,10 +256,13 @@ def create(self, name, url, description=None, scm_credentials=None, scm_branch=N
             delete_on_update=delete_on_update,
             update_on_launch=update_on_launch)
 
-        wait_for(lambda: repository.exists,
-                 fail_func=repo_list_page.browser.selenium.refresh,
-                 delay=5,
-                 timeout=900)
+        wait_for(
+            lambda: repository.exists,
+            fail_func=repo_list_page.selenium.refresh,
```

Comment:
Why is selenium refresh being called here, refresh should be called through browser, like it is in the view below.

I would also keep the delay at 5s.

Reply:
@mshriver  done